.travis.yml

```diff
@@ -1,17 +1,36 @@
 ---
 sudo: false
+dist: xenial
 language: ruby
-#cache: bundler
 rvm:
   - 2.2.4
-before_install: 
-  - gem install bundler -v '1.17'
+matrix:
+  include:
+    - env: OPENSTUDIO_VERSION=2.8.0 && OPENSTUDIO_SHA=55665635f0 && RUBYLIB=/usr/local/openstudio-2.8.0/Ruby:/usr/Ruby
+before_install:
   - gem install bundler -v '~> 1.17'
+  - sudo rm /usr/local/bin/docker-compose
 install: 
   - bundle install
+before_script:
+  - curl -sLO https://raw.githubusercontent.com/NREL/OpenStudio-server/develop/docker/deployment/scripts/install_openstudio.sh
+  - chmod +x install_openstudio.sh
+  - sudo ./install_openstudio.sh $OPENSTUDIO_VERSION $OPENSTUDIO_SHA
 script:
-  - ls
-  - rspec spec/tests/selection_tool_spec.rb
-#  - bundle exec rake 
-#  - bundle exec rake openstudio:test_with_openstudio
-#  - bundle exec rake   
\ No newline at end of file
+  - bundle exec rspec spec/tests/selection_tool_spec.rb
```

Comment:
It appears that there a few tests that fail if we include all the tests. I think that is okay for now, but we need to address those. This line should just be

```
bundle exec rake
```

which will run all of the tests.

Reply:
all test are running. The issue is that we cannot run all of them since the simulating all scenarios will take too much time for travis testing.